test/CarouselSpec.js

```diff
@@ -223,6 +223,84 @@ describe('<Carousel>', () => {
     }, 150);
   });
 
+  it('should call onSlide when slide animation is disabled', () => {
+    const clock = sinon.useFakeTimers();
```

Comment:
**suggestion:** might be cleaner to wrap these all in a `describe` and set up / tear down sinon in `beforeEach` / `afterEach`

Reply:
Done!  Thanks